source/gfoidl.Base64/Internal/Base64UrlEncoder/Base64UrlEncoder.Encode.cs

```diff
@@ -231,8 +231,32 @@ private static int GetNumBase64PaddingCharsAddedByEncode(int dataLength)
             // 0 -> 0
             // 1 -> 2
             // 2 -> 1
+            int mod3 = FastMod3(dataLength);
+            return mod3 == 0 ? 0 : (3 - mod3);
+        }
+        //---------------------------------------------------------------------
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int FastMod3(int value)
```

Comment:
Ah shit, keep the `uint` please.

Reply:
working on it!

Reply:
> keep the `uint` please.

This one is left, otherwise the assert will be useless. Please revert that, then it good to merge.